phpstan.neon.dist

```diff
@@ -0,0 +1,9 @@
+parameters:
+    level: 1
```

Comment:
This sounds like a really painfully low analysis level: is it maybe possible to use psalm with aggressive configuration and a baseline instead? That would at least prevent new bugs from sneaking through.

Reply:
Yeah, I would like to do that. Im just taking it slow with one PR at the time.

Reply:
Fair enough - my idea was to replace phpstan for this kind of projects: no baseline means very ineffective static analysis

Reply:
@Ocramius there's [baseline for PHPStan](https://medium.com/@ondrejmirtes/phpstans-baseline-feature-lets-you-hold-new-code-to-a-higher-standard-e77d815a5dff) now too. Is this still valid?

> my idea was to replace phpstan for this kind of projects

Reply:
Yeah, my comment was put in before I knew phpstan added baseline support: I suggest going with `--level=max` and a baseline config.